### PR TITLE
gassi.csr

### PR DESCRIPTION
**Pull request by matthieuleroyer**
@MorrisJobke

### CODE REVIEW

**Comment by MorrisJobke** [comment]
@matthieuleroyerexternal Do you have the app somewhere available as source code? Just want to check the appid. And just out of curiosity: what does this app do? 😃

File: gassi/gassi.csr [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+9thFnUetvXgJQ0qiYwBXkOOllwZCGcQchUy8U/LWA3ctegN205IpsNyJqGnDGhId
+s8M3Zed0/ebqMwsZPqzeKgcW3gm18ftWED7dwH6Yqjd+aJjNJY/8BYnMnZy7IfGg
+TkclJ8NJD6RSFERuak28+VgUZNPOPtHcuKSCL3e17Ok1wdOSyPzoD7bexA0xzEP6
+/t1hZFZZ7j1jM2jva3uGxPgWPCRRKPqxsBP4fTenuros9yMi8xSXhqobYnoqoezi
+xbOFkNkuWTEBVY0TKNfvdFQGSQHN1Ekm5Bcgra9GoyZKlIgKrpK+xu0/VxHq18tA
+Ghgp9+HW8921I5QwK8DVRqPpGb/eP+mGh5C2B5JMQXlaT7BGNTdlmsrKowbNnYeQ
+CJHBrtA3rlwkkffNw7pGgFSKMstIStwCR1zrDqjMU+T8F0Xo/JBQX6r9/b7JRVlJ
+Hu3V5xTXxq0sDQnskW/8e42WcjrZhqnK/wx3rLK2stlZTGCTt/ZPzHweX/2cndz/
+9VpeEsUKnnvH
+-----END CERTIFICATE REQUEST-----
```
Comment (516300502) [inline gassi/gassi.csr]: -----BEGIN CERTIFICATE REQUEST-----
MIIEVDCCAjwCAQAwDzENMAsGA1UEAwwEbmV3czCCAiIwDQYJKoZIhvcNAQEBBQAD
ggIPADCCAgoCggIBAONu7I+JYH/8/oCE/QaCpldz20SpnwJPJ8wmKzargYMITif2
u/NF5bCRQzrSrziIO73PyQIpN+7G3sUtzend0aMIGcgwP7+ShoLDqIK1adhVU2TY
2ij2DqRJK0NuUGgPgXgu30tn2ZdItGYnFxDjw4tSRKOWln+80zXcMk7LBAlN34hM
ubTwa+Vw3LG6HrAbLBSiMVCfHOEAseSFzoalSV+qP0qrtel/ByFrP5Oup5d7hB3g
88iWTxqCgzjI38oahj+dPADH+7EL0LKYV0844x4TJOfxqh2KbbsiX4A+nzMovPwA
K2w7CZgd6CeC3XlvnhrKGSauXEABsNNUZBlh6BORb60ReFEeUMti7+BcSoBKXAlO
MRjaJ3FE7riZ5iZFHdN0zSVcs2GKsIELTbGlumwXPnBuEK53gmlm6lIIoYKOG+OW
vWEEsyCSc2stHD3iRyi1iciqHzipnR9WLJpdKuXT/gmhZzwmKwoYPb07DbsJFwFP
L6dkmJ950c0dAvPEVRRXl+Ho0b8w4WsiVGuDddjcWqENM6UyzKoOxwJdTW3wJXTA
kBFC9M8VVghXv9/1zJrViIItwxRAM3IkDezwuz1k1620ktHjpQ0KSvCtvFHX5laf
OiIgvkIqlAPI9yAVpzu9ZvsJVtdDEmZW47ZBECVqNufO15s6YtlWHdyVIowvAgMB
AAGgADANBgkqhkiG9w0BAQsFAAOCAgEAW23se48KsUO1bYmovLyQwak8PnmZEbVj
ubBIBrutJRjt5KYJ2YL5VouhvFZtcXAdBHTljzXhsxyRSzyy6M8pgePJ62Ut/OVs
efMdWCAk7WJ+E1p+bibzd8qSByQ7VRHrbjxUlnpO/8jFXnY4WZkkrCKbb5unjzjr
Vi2c83PI4YIIMvi0BRSgpH4uI/qtg5P+3AKTELpMTiekUBf4msDibMosfDjzyTva
MAlIscxuQXjjC+Td/ZUITVdlTfE6/0Qu266ooJ/SKArwMxDpHF9Ptul2/6Yew1Sl
KqHU0bAUWSoTZR/GLYg69Xl1sy0i2H0gkxzW+MzFRN0BXFM0+tMc7Od6EoNyOJ7i
zrHeRhu8Mq/0XfMq2stUBFAgQCfiGf/1q+slX2M7YZZi5UymlVZpljQMszdL1Jqy
iYpudcmzFeSrRFZfGj40X83Zwxz3dd3q5Jru+D36uscv6DX0iNhtmp9nCYVlOaxw
tVpIH90TbY+KeFA8qhGPqPI0eZw1lW2hgTOaclOraygM9P5r95+jDsHWLmuenSJG
20qgqWED0onS9RqMcwpekvaUa/IcBN1MyQMPBxP0u/ROPZAUT4vIktRGA2gu9YLz
U3sLe00337OzgdolapNetcA6ZdObU29vmNbYpqOVn7KUEyqmgOtMCH4B+8XnEDMJ
EKIA8mRN54w=
-----END CERTIFICATE REQUEST-----

Comment (MorrisJobke) [reply gassi/gassi.csr]: If you want to change the CSR then please update the PR accordingly. If you need help with this just raise your questions and we are happy to help. :)

**Comment by jancborchardt** [comment]
@matthieuleroyerexternal @516300502 any update here? Please include a link to the repository which should have a short description and a screenshot. :)

**Comment by MariusBluem** [comment]
@matthieuleroyerexternal ping

**Comment by matthieuleroyerexternal** [comment]
Hi,
sorry for the delay, the project has been canceled, so you can delete this certificate

thanks in advance,

best regards,

Matthieu

**Pull request closed without merging**